monai/deploy/core/models/torch_model.py

```diff
@@ -47,7 +47,9 @@ def predictor(self) -> "torch.nn.Module":  # type: ignore
             torch.nn.Module: the model's predictor
         """
         if self._predictor is None:
-            self._predictor = torch.jit.load(self.path).eval()
+            # If GPU is not available, explicitly use "cpu" to dynamically remap storages to CPU.
+            map_location = None if torch.cuda.is_available() else "cpu"
```

Comment:
Thanks for the suggestion!

I am afraid the behavior is different (map_location needs to be `None`, not torch.device("cuda")).

We wanted to remap to 'cpu' only if cuda is not available. If cuda is available, we wouldn't want to remap location, and use the device specified in the model.

```python
device = torch.device("cuda" if torch.cuda.is_available() else "CPU")
self._predictor = torch.jit.load(self.path, map_location=device).eval()
```
vs
```python
map_location = None if torch.cuda.is_available() else "cpu"
self._predictor = torch.jit.load(self.path, map_location=map_location).eval()
```

However, on second thought, it would be better to force remap, depending on cuda availability (as operator/application's code also usually use the condition to determine which device to use.

So, will fix as suggested